packages/e2e/integration/athlete_self_register.ts

```diff
@@ -91,4 +91,39 @@ describe("Athlete self registration", () => {
       t(Alerts.ContactEmail, { email: __emailFrom__ }).replace(/<\/?a.*>/g, "")
     );
   });
+
+  it("checks for a spinner on the submit button while the submit request is processing", () => {
```

Comment:
it "shows a spinner while the submit request is being processed". `it` being the code you're testing, not the test itself.

Reply:
> "It puts the lotion in the basket"
😂

Reply:
![image](https://github.com/eisbuk/EisBuk/assets/396425/8317c488-7e7d-47df-b059-29eea5e78c7f)